datausage.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "U.S. Greenhouse Gas Center: Data Usage Notebooks"
+---
+
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data usage notebooks, your gateway to exploring and analyzing curated datasets on greenhouse gas emissions.
+Our cloud-based system offers seamless access to GHG curated datasets. Dive into the data with our data usage Jupyter notebooks, which demonstrate how to explore, access, visualize, and conduct basic data analysis for each GHG Center dataset in a code notebook environment. The data usage notebooks are grouped topically. The dataset product type (model output, satellite observation, etc.) is noted next to the notebook name. Click on a notebook to learn more about the dataset and to view the data usage code.
+
+Join us in our mission to make data-driven environmental solutions.
+Explore, analyze, and make a difference with the US GHG Center.
+
+[View the US GHG Center Data Catalog](https://earth.gov/ghgcenter/data-catalog)
+
+## Gridded Anthropogenic Greenhouse Gas Emissions
+1. OCO-2 MIP Top-Down CO₂ Budgets - Model Output
```

Comment:
Recommend formatting all usage notebooks as following - list the dataset name and then add a sub-bullet (or sub-bullets if the dataset has multiple notebooks) that link to the individual notebooks. Also remove the product type since I don't think it adds much useful context. E.g.

1. OCO-2 MIP Top-Down CO₂ Budgets
- [Beginner level notebook](user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb) to access, visualize, explore statistics, and create a time series of the OCO-2 MIP Top-Down CO₂ Budgets dataset
-  [Intermediate level notebook to read and visualize](user_data_notebooks/oco2-mip-National-co2budget.ipynb) National CO₂ Budgets using OCO-2 MIP Top-Down CO₂ Budget country total data. This notebook utilizes the country totals available at [ceos.org/gst/carbon-dioxide](https://ceos.org/gst/carbon-dioxide.html), which compliment the global 1° x 1° gridded CO₂ Budget [data featured in the US GHG Center](https://earth.gov/ghgcenter/data-catalog/oco2-mip-co2budget-yeargrid-v1).   
2. ODIAC Fossil Fuel CO₂ Emissions
- [Beginner level notebook](user_data_notebooks/odiac-ffco2-monthgrid-v2022_User_Notebook.ipynb) to access, visualize, explore statistics, and create a time series of the ODIAC Fossil Fuel CO₂ Emissions dataset

Etc.